pctasks/core/pctasks/core/storage/blob.py

```diff
@@ -388,7 +393,8 @@ def get_path(self, uri: str) -> str:
             return blob_uri.blob_name or ""
 
     def get_file_info(self, file_path: str) -> StorageFileInfo:
-        with self._get_client() as client:
+        client = self._get_client()
+        with contextlib.nullcontext():
```

Comment:
This is just to minimize the diff. We can also just dedent everything under here, or we can remove `ContainerClientWraper.__exit__`